community/hangouts/index.md

```diff
@@ -39,29 +33,21 @@ meta_descr: hangouts.intro
     <section class="container">
           <div class="row">
                 <!-- left two-thirds block-->
-               <div class="left two-thirds col-lg-8 col-md-8 col-sm-12 col-xs-12">
-                <div class="col-xs-12">
-                        <div class="info-block">
+               <div class="left one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                <div class="col-xs-12 info-block">
                             <div class="row center-xs">
                                 <div class="col">
                                     <h2>{% t hangouts.resources %}</h2>
                                 </div>
                             </div>
                             <div class="row around-xs">
                                 <div class="center-xs">
-                                    <p>{% t hangouts.resources_para %} {% t hangouts.resources_para1 %}</p>
-                                </div>
-                                <div class="col">
-                                    <p><a href="https://translate.getmonero.org/" class="btn-link btn-fixed">Weblate</a></p>
-                                </div>
-                                <div class="col">
-                                    <p><a href="https://matrix.to/#/%23monerospace:monero.social" class="btn-link btn-fixed">Matrix</a></p>
```

Comment:
matrix monerospace is a great catch-all place for matrix users interested in monero, not sure why its being removed

Reply:
yes by the matrix mods